docs/reference/operations-guide/export.md

```diff
@@ -0,0 +1,1949 @@
+# Export Operations
+
+These Operations are used for exporting... They are not always available - depends on the operartion.
```

Comment:
```suggestion
These Operations are used for exporting... They are not always available - depends on the operation.
```

Reply:
This whole thing should have been replaced with an actual description. Thanks for spotting this!